Week3/homework/2-step3.js

```diff
@@ -3,6 +3,25 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const values = [];
   // Add your code here
+
+  for (startIndex; startIndex <= stopIndex; startIndex++) {
+    values.push(startIndex);
+  }
+  console.log('Array created is: [' + values + ']');
```

Comment:
Nicely done.  And here is a fun brain teaser, that you might enjoy. 
This is how I created the starting array to iterate over:
```
const values = [...Array((stopIndex + 1) - startIndex).keys()].map(i => i + startIndex);
```
Something to puzzle over in your spare time. 😉 
In many languages creating an array with index values is often called `range()`.  Javascript, unfortunately, does not have this, and this was the only one-liner I could find.